apps/src/lab2/levelEditors/predictSettings/MultipleChoiceFields.tsx

```diff
@@ -59,6 +59,7 @@ const MultipleChoiceFields: React.FunctionComponent<
     return null;
   }
 
+  const solutions = predictSettings.solution?.split(',') || [];
```

Comment:
There was a bug here that made it so that we'd automatically check boxes for options where the string included another option as a substring:

![2025-03-06 16-45-03 2025-03-06 16_45_37](https://github.com/user-attachments/assets/2526d8a3-f605-4c26-80df-54413a0c0e36)